### DISCUSSION TITLE
Wired Router

### DISCUSSION THREAD

**Discussion by pardal12** [Koenkk/zigbee2mqtt#22725] [Devices]
First of all excuse me for my English.
I’ve been searching but haven’t been able to find a thread similar to what I need.
I have a local that does not reach zigbee coverage and I only have 1 ethernet cable connected to the home network.
Is there any option to install a zigbee router that can transmit signals to Home Assistant via network cable?

Thank you

**Comment by francisp2** [comment]
You can't transport Zigbee signals over Ethernet, so no chance you will one.
